composer.json

```diff
@@ -42,8 +43,8 @@
         "symfony/http-kernel": "~2.3",
         "symfony/process": "~2.1",
         "symfony/yaml": "~2.1",
-        "twig/twig": "~1.9",
```

Comment:
In this reordering you lost track of the commas. We need a comma after twig/extensions now and removed from after twig/twig.